server/app/services/geo/esri/EsriClient.java

```diff
@@ -60,10 +69,19 @@ public EsriClient(Config configuration, WSClient ws) {
         configuration.hasPath("esri_find_address_candidates_url")
             ? Optional.of(configuration.getString("esri_find_address_candidates_url"))
             : Optional.empty();
-    this.ESRI_FIND_ADDRESS_CANDIDATES_TRIES =
-        configuration.hasPath("esri_find_address_candidates_tries")
-            ? configuration.getInt("esri_find_address_candidates_tries")
+    this.ESRI_EXTERNAL_CALL_TRIES =
+        configuration.hasPath("esri_external_call_tries")
+            ? configuration.getInt("esri_external_call_tries")
             : 3;
+    this.esriServiceAreaValidationConfig = new EsriServiceAreaValidationConfig(configuration);
```

Comment:
Updating `isAddressLocationInServiceArea` to accept `EsriServiceAreaValidationOption` instead of a string, removed the need for `EsriServiceAreaValidationConfig`.